tests/utils/pvc.go

```diff
@@ -0,0 +1,87 @@
+package utils
+
+import (
+	"fmt"
+	"time"
+
+	k8sv1 "k8s.io/api/core/v1"
+	apierrs "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+)
+
+const (
+	DefaultPvcMountPath = "/pvc"
+	defaultPollInterval = 2 * time.Second
+	PVCCreateTime       = 30 * time.Second
+	PVCDeleteTime       = 30 * time.Second
+	PVCWaitForTime      = 30 * time.Second
```

Comment:
Even more specific -- what do you think of `PVCPhaseTime = ...`

Reply:
We already have create and delete, why not phase. I will rename it.